octobot_services/managers/service_feed_manager.py

```diff
@@ -0,0 +1,31 @@
+#  Drakkar-Software OctoBot-Services
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+from octobot_services.service_feeds.abstract_service_feed import AbstractServiceFeed
+
+
+class ServiceFeedManager:
+
+    @staticmethod
+    async def start_service_feed(service_feed: AbstractServiceFeed, backtesting_enabled: bool):
+        if not service_feed.is_running and not service_feed.should_stop:
+            if await service_feed.initialize(backtesting_enabled):
+                service_feed.start()
```

Comment:
Can we imagine to `return service_feed.start()` with `service_feed.start` that returns a boolean correponding to the real starting status.